pages.fr/linux/userdel.md

```diff
@@ -1,8 +1,17 @@
 # userdel
 
-> Efface un utilisateur.
+> Supprime un compte utilisateur.
+> Voir aussi `users`, `useradd`, `usermod`.
 > Plus d'informations : <https://manned.org/userdel>.
 
-- Efface un utilisateur et son dossier home :
+- Supprime un utilisateur :
 
-`userdel -r {{nom}}`
+`sudo userdel {{nom_utilisateur}}`
+
+- Supprime un utilisateur dans un autre répertoire racine :
+
+`sudo userdel --root {{chemin/vers/autre/racine}} {{nom_utilisateur}}`
```

Comment:
```suggestion
`sudo userdel --root {{chemin/vers/autre_racine}} {{nom_utilisateur}}`
```

Optional fix (the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/userdel.md) page can be updated to reflect this change): `_` can be used instead of `/`.